internal/git/gitsync_processor.go

```diff
@@ -127,14 +132,24 @@ func watchRepo(ctx context.Context, r *git.Repository, gitSync *v1alpha1.GitSync
 		err = tree.Files().ForEach(func(f *object.File) error {
 			logger.Debugw("read file", "file_name", f.Name)
 			// TODO: this currently assumes that one file contains just one manifest - modify for multiple
-			manifest, err := f.Contents()
-			if err != nil {
-				logger.Errorw("cannot get file content", "filename", f.Name, "err", err)
-				return err
-			}
+			// TODO :need to address the valid file extension issue
```

Comment:
do we still need this TODO comment?

Reply:
actually, even the one above has been addressed, right? I think we can remove both